src/main/java/org/sagebionetworks/bridge/services/AccountWorkflowService.java

```diff
@@ -208,8 +217,9 @@ public void sendEmailVerificationToken(Study study, String userId, String recipi
         }
 
         String sptoken = getNextToken();
+        long expiresOn = getDateTimeInMillis() + (VERIFY_OR_RESET_EXPIRE_IN_SECONDS*1000);
 
-        saveVerification(sptoken, new VerificationData(study.getIdentifier(), ChannelType.EMAIL, userId));
+        saveVerification(sptoken, new VerificationData(study.getIdentifier(), ChannelType.EMAIL, userId, expiresOn));
```

Comment:
Do we have any unit tests that verify that we're saving the expiresOn to the cacheProvider correctly? (For both email and phone case.)